service-node/src/main/scala/org/enterprisedlt/fabric/service/node/FabricNetworkManager.scala

```diff
@@ -199,7 +199,7 @@ class FabricNetworkManager(
                       instantiateProposalRequest.setChaincodeLanguage(nodeType)
                       Right(())
 
-                  case _ => Left(s"Wrong cc lang type")
+                  case _ => Left("Wrong cc lang type")
```

Comment:
pls rename: Wrong -> Unsupported
also would be good to add (to the message) the value was passed